### PR TITLE
Add slashing protection CLI options.

### PR DESCRIPTION
**Pull request by bgravenorst**
Signed-off-by: Byron Gravenorst <byron.gravenorst@consensys.net>

## Pull Request Description
Add slashing protection CLI options.

### CODE REVIEW

File: docs/HowTo/Prevent-Slashing.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+When importing the slashing-protection file, Teku imports the file to the
+`<data-path>/validators/slashprotection/` directory in the format `<validator-pubkey>.yml`
+(with no 0x prefix).
+
+Teku supports files using the [Minimal] or [Complete] interchange format when importing.
+
+!!! example
+
+    ```bash
+    teku slashing-protection import --data-path=/home/me/me_node --from=/home/slash/validatorNode1.yml
```
Comment (rolfyone) [inline docs/HowTo/Prevent-Slashing.md:24]: importing is from the interchange format, which is json and for potentially a large number of validators in the one import file... `--from=/home/slash/slashing-interchange-format.json`

File: docs/HowTo/Prevent-Slashing.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+## Exporting a slashing-protection file
+
+Export the slashing-protection file when migrating a validator to a different Teku, or non-Teku
+node. Teku exports the slashing protection file in the [Minimal] format.
+
+!!! example
+
+    ```bash
+    teku slashing-protection export --data-path=/home/me/me_node --to=/home/slash/validatorNode1.yml
```
Comment (rolfyone) [inline docs/HowTo/Prevent-Slashing.md:44]: output is json format, and while you could name it a .yml file, that'd be super confusing :)

`--to=slashing-interchange-format-minimal.json`

File: docs/Reference/CLI/CLI-Subcommands.md [hunk 3]
```diff
@@ -715,3 +715,99 @@ Validators public withdrawal key.
...
+=== "Syntax"
+
+    ```bash
+    teku slashing-protection import --from=<FILE>
+    ```
+
+=== "Command Line"
+
+    ```bash
+    teku slashing-protection import --from=/home/slash/b845...23bed.yml
```
Comment (rolfyone) [inline docs/Reference/CLI/CLI-Subcommands.md:758]: again, from would be json

File: docs/Reference/CLI/CLI-Subcommands.md [hunk 4]
```diff
@@ -715,3 +715,99 @@ Validators public withdrawal key.
...
+=== "Syntax"
+
+    ```bash
+    teku slashing-protection export --to=<FILE>
+    ```
+
+=== "Command Line"
+
+    ```bash
+    teku slashing-protection export --to=/home/slash/b845...23bed.yml
```
Comment (rolfyone) [inline docs/Reference/CLI/CLI-Subcommands.md:803]: to will be json

**Review by rolfyone — APPROVED** [review]
minor comments. LGTM overall

File: docs/HowTo/Prevent-Slashing.md (hunk 1 above)
Comment (bgravenorst) [reply docs/HowTo/Prevent-Slashing.md:24]: Done

File: docs/HowTo/Prevent-Slashing.md (hunk 2 above)
Comment (bgravenorst) [reply docs/HowTo/Prevent-Slashing.md:44]: Done

File: docs/Reference/CLI/CLI-Subcommands.md (hunk 3 above)
Comment (bgravenorst) [reply docs/Reference/CLI/CLI-Subcommands.md:758]: done

File: docs/Reference/CLI/CLI-Subcommands.md (hunk 4 above)
Comment (bgravenorst) [reply docs/Reference/CLI/CLI-Subcommands.md:803]: Done

**Pull request merged**